src/main.cpp

```diff
@@ -143,10 +143,17 @@ struct JsonFileFinder
             return;
         }
 
+        // convert windows new-lines to UNIX byt removing
+        // all instances of '\r'
+        std::string markerCheckStr = str;
```

Comment:
Maybe overly paranoid to make a copy before modifying. Not sure if anything in the time profiler would ever output valid \r characters in JSON string literals.